spec/vast_trackers.spec.js

```diff
@@ -0,0 +1,147 @@
+import { VASTClient } from '../src/vast_client';
+import { VASTParser } from '../src/parser/vast_parser';
+import { VASTTracker } from '../src/vast_tracker';
+import { inlineTrackers } from '../spec/samples/inline_trackers';
+
+const vastParser = new VASTParser();
+const vastClient = new VASTClient();
+
+describe('VASTTracker', function() {
+  describe('#constructor', function() {
+    let vastTracker = null;
+    let response = {};
+
+    beforeAll(done => {
+      const vastXml = new window.DOMParser().parseFromString(
+        inlineTrackers,
+        'text/xml'
+      );
+      vastParser.parseVAST(vastXml).then(res => {
+        response = res;
+        done();
+      });
+    });
+
+    describe('#linear', () => {
+      let spyEmitter;
+      let spyTrackUrl;
+      beforeAll(() => {
```

Comment:
IMO this should be a `beforeEach` to avoid unit tests impacting other unit tests